GenderPayGap.WebUI/Views/Register/ChooseOrganisation.cshtml

```diff
@@ -67,7 +67,32 @@
                                     <div class="bold-small">Showing @Model.Employers.VirtualRecordTotal.ToString("N0") of @Model.Employers.ActualRecordTotal.ToString("N0") results. Please refine your search or enter your company number.</div>
                                 </div>
                             }
-                            <p style="width: 66%">Please choose your organisation and we'll send an activation PIN to its registered address. Only organisations registered at Companies House are listed here.</p>
+                            
+                            if(!FeatureFlagHelper.IsFeatureEnabled(FeatureFlag.PrivateManualRegistration))
+                            {
+                                <div class="column-two-thirds">
+                                    <p>
+                                        Please choose your organisation and we'll send an activation PIN to its 
+                                        registered address. Only organisations registered at Companies House 
+                                        are listed here.
+                                    </p>
+                                </div>

```

Comment:
This is the old copy and we still want to see it in the case where the feature flag is turned off.